src/helix/client/client_ext.rs

```diff
@@ -1202,6 +1202,79 @@ impl<'client, C: crate::HttpClient + Sync + 'client> HelixClient<'client, C> {
             vec
         })
     }
+
+    #[cfg(feature = "eventsub")]
+    /// Get all [Conduits](crate::eventsub::Conduit) for this [Client](twitch_oauth2::TwitchToken)
```

Comment:
```suggestion
    /// Get all [Conduits](crate::eventsub::Conduit) for this [Client](crate::Client)
```

This is wrong in the method above too.

Reply:
Yeah wording is a bit weird here - the Client I refer to here is a Twitch Developer Application, so I thought Token might be ok. Maybe I should just write exactly that instead of linking to Client

Reply:
Oh right, that's actually correct but should probably be reworded.

Reply:
The latter seems reasonable yeah